include/mgp.hpp

```diff
@@ -4279,9 +4279,90 @@ inline void AddParamsReturnsToProc(mgp_proc *proc, std::vector<Parameter> &param
 }
 }  // namespace detail
 
+inline bool CreateLabelIndexImpl(mgp_graph *memgaph_graph, const std::string_view label) {
+  return CreateLabelIndex(memgaph_graph, label);
+}
+
+inline bool DropLabelIndexImpl(mgp_graph *memgaph_graph, const std::string_view label) {
+  return DropLabelIndex(memgaph_graph, label);
+}
+
+inline List ListAllLabelIndicesImpl(mgp_graph *memgraph_graph) {
+  auto *label_indices = mgp::MemHandlerCallback(ListAllLabelIndices, memgraph_graph);
+  if (label_indices == nullptr) {
+    mgp::list_destroy(label_indices);
```

Comment:
If label_indices is nullptr, why call destroy on list?

Reply:
yes you are right. I used that because  I saw it in other function but it is not needed